src/main/oAuthClient.ts

```diff
@@ -276,6 +286,52 @@ export default class OAuthClient {
     })
   }
 
+  loginWithIdToken(provider: string, idToken: string, nonce: string, opts: AuthOptions = {}): Promise<void> {
+    const authParams = this.authParams({
+      ...opts,
+      useWebMessage: false
+    })
+
+    return this.loginWithRedirect({
+      provider,
+      idToken,
+      nonce,
+      ...authParams
+    })
+  }
+
+  googleOneTap(opts: AuthOptions = {}) {
+    if (!isUndefined(this.config.googleClientId)) {
```

Comment:
What's the error handling in the SDK Core that could fit in the else of this structure?

Reply:
Can use `if (this.config?.googleClientId)`

Reply:
Depends if you want the error in the console or the API response.
`logError` for console, `Promise.reject(new Error(...))` for API response.
Means you have to return a promise in the nominal case as well.